README.md

```diff
@@ -138,34 +129,23 @@ addresses are provided when creating "applicants". To search for addresses
 by postcode, use:
 
 ```java
-onfido.address.pick("SE1 4NG")
-```
-
-#### Webhook Endpoints
-
-Onfido allows you to set up and view your webhook endpoints via the API, as well
-as through the dashboard.
-
-```java
-onfido.webhook.create(requestBody)        # => Creates a webhook endpoint
-onfido.webhook.find("webhook_id")         # => Finds a single webhook endpoint
-onfido.webhook.list()                     # => Returns all webhook endpoints
+onfido.address.pick(<POSTCODE_HERE>)
 ```
 
 #### SDK Tokens
 
 Onfido allows you to generate JSON Web Tokens via the API in order to authenticate
-with Onfido"s [JavaScript SDK](https://github.com/onfido/onfido-sdk-ui).
+with Onfido"s [Web SDK](https://github.com/onfido/onfido-sdk-ui).
 
 ```java
-onfido.sdk_token.create(applicant_id: "applicant_id", referrer: "referrer")
+onfido.sdk_token.create(applicant_id: <APPLICANT_ID_HERE>, referrer: "referrer")
```

Comment:
Minor: could we put `referrer: "<REFERRER_HERE`">? Or alternatively:

`"referrer": "https://*.example.com/example_page/**"`